debian/control

```diff
@@ -0,0 +1,13 @@
+Source: revpi-bluetooth
+Section: misc
+Priority: optional
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Build-Depends: debhelper-compat (= 12)
+Standards-Version: 4.3.0
+Homepage: https://www.kunbus.com
```

Comment:
Add the github page as Vcs-Browser and Vcs-Git
https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-vcs-fields